.github/workflows/GAP.yml

```diff
@@ -0,0 +1,26 @@
+name: "GAP tests"
+
+# Trigger the workflow on push or pull request
+on:
+  - push
+  - pull_request
+
+jobs:
+  test:
+    name: "GAP ${{ matrix.gap-branch }} on ${{ matrix.os }}"
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [ubuntu-latest]
+        gap-branch:
+          - master
+          - stable-4.11
```

Comment:
You _should_ be able to add `stable-4.10` here with no extra work. The reason I haven't done this yet for Digraphs is because there's a tiny bit of extra effort that will be needed to install NautyTracesInterface (which is optional, and not needed here).